src/nora.nim

```diff
@@ -11,13 +11,14 @@ import
     qqmlcontext, qurl, qobject, qvariant, qmetatype, qmetaproperty, qstringlistmodel,
     qeventloop,
   ],
-  seaqt/QtCore/gen_qnamespace,
+  seaqt/QtCore/[gen_qnamespace, qtcore_pkg],
   ./nimside
 
 const
   curPath = currentSourcePath.parentDir
-  rccPath = gorge("pkg-config --variable=libexecdir Qt6Core") & "/rcc"
-  cflags = gorge("pkg-config --cflags Qt6Core")
+  qtMajor = QtCoreGenVersion.split(".")[0]
+  rccPath = gorge("pkg-config --variable=host_bins Qt" & qtMajor & "Core") & "/rcc"
```

Comment:
huh, weird - on fedora, the supplied pkg-config file uses `libexecdir` for qt6 - is this an outcome of the prl-to-pc step, that a different variable is used?

Reply:
ok - I just pushed a workaround which tries both :) qmake or prl-to-pc, any seems fine .. might be worth considering the ios build.

Reply:
Ohh, the IOS builds are a different beast..The prl-to-pc isn't enough there. On IOS we have lots of Frameworks instead of libs + prl files and all libs are static. So we need to work with much less info.
I've used a modified prl-to-pc to generate data for most libs and frameworks, but with some libs I've needed to manually fix the pc file.

I was thinking to generate an initial version of the pc files and then to manually maintain these files - to drop the prl-to-pc and replace it with a repo that serves these pc files.

Reply:
> repo that serves these pc files.

they contain installation-local paths though, no ..?

ie another weakness  of .pc files in general is their poor windows support, also something to consider